File: maxmina/maxmina.c
#include <stdio.h>

int main()
{
    int case_num;
    scanf("%d", &case_num);
    for (int i = 0; i < case_num; i++)
    {
        int n, k;
        scanf("%d%d", &n, &k);
        int num;
        int yes = 0;
        for (int j = 0; j < n; j++)
        {
            scanf("%d", &num);
            if (num == 1)
            {
                yes = 1;
            }
        }

        if (yes)
        {
            printf("YES\n");
        }

        else
        {
            printf("NO\n");
        }
    }
}

File: maxmina/maxmina.py
def main():
    case_num = int(input())
    for _ in range(case_num):
        n, k = map(int, input().split())
        arr = list(map(int, input().split()))
        yes = False
        for num in arr:
            if num:
                yes = True
        if yes:
            print("YES")
        else:
            print("NO")
    
if __name__ == "__main__":
    main()
